loadflow/loadflow-validation/src/main/java/com/powsybl/loadflow/validation/TransformersValidation.java

```diff
@@ -79,7 +79,7 @@ public boolean checkTransformer(TwoWindingsTransformer twt, ValidationConfig con
         double rho = ratioTapChanger.getCurrentStep().getRho();
         double rhoPreviousStep = tapPosition == lowTapPosition ? Double.NaN : ratioTapChanger.getStep(tapPosition - 1).getRho();
         double rhoNextStep = tapPosition == highTapPosition ? Double.NaN : ratioTapChanger.getStep(tapPosition + 1).getRho();
-        double targetV = ratioTapChanger.getTargetV();
+        double regulationValue = ratioTapChanger.getRegulationValue();
```

Comment:
revert to using targetV variable name, set to NaN if regulation mode is not voltage